packages/SwingSet/tools/vat.js

```diff
@@ -45,6 +45,8 @@ async function main() {
       deepLog(d.promises);
       console.log('Run Queue:');
       deepLog(d.runQueue);
+      console.log('Acceptance Queue:');
```

Comment:
Oh, how nostalgic, I'd completely forgotten about this file. Some day I look forward to defining a `/usr/bin/vat` in the way this originally aspired to. But I bet Endo will beat me to it by a mile.